scripts/update-product-version.ps1

```diff
@@ -20,6 +20,17 @@ if ("$majorVersion.$minorVersion.$revisionNumber" -match '^\d+\.\d+\.\d+$') {
         $newcontent | Set-Content $ebpf_version_file -NoNewline
         Write-Host -ForegroundColor DarkGreen "Version number updated to '$majorVersion.$minorVersion.$revisionNumber' in $ebpf_version_file"
 
+        # Update the version.json file as well
```

Comment:
```suggestion
        # Update the version.json file as well.
```
Comment style consistency (though it seems not all comments are consistent in this file right now, but most are)